catroid/src/main/java/org/catrobat/catroid/ui/ProjectActivity.java

```diff
@@ -94,6 +92,10 @@
 
 	public static final String EXTRA_FRAGMENT_POSITION = "fragmentPosition";
 
+	public static final int REQUEST_CODE_VISUAL_PLACEMENT_NEW_SPRITE = 2121;
+	public static final String EXTRA_X_TRANSFORM_NEW_SPRITE = "X";
+	public static final String EXTRA_Y_TRANSFORM_NEW_SPRITE = "Y";
```

Comment:
Please use `EXTRA_X_TRANSFORM` and `EXTRA_Y_TRANSFORM` from the SpriteActivity.

Reply:
Done.